test/unit/fixture/reverse_with_layer_filtering_non_coarse_subset.js

```diff
@@ -0,0 +1,41 @@
+var vs = require('../../../query/reverse_defaults');
+
+module.exports = {
+  'query': {
+    'bool': {
+      'filter': [
+        {
+          'geo_distance': {
+            'distance': '500km',
```

Comment:
The plan is to remove the initial coarse reverse (we discussed this in standup) since non-coarse reverse now is limited to just non-coarse `layers` scenearios.